src/openrct2-ui/windows/GuestList.cpp

```diff
@@ -490,7 +492,7 @@ static void window_guest_list_scrollgetsize(rct_window* w, int32_t scrollIndex,
             }
             w->var_492 = numGuests;
             y = numGuests * SCROLLABLE_ROW_HEIGHT;
-            _window_guest_list_num_pages = (int32_t)std::ceil((float)numGuests / 3173);
+            _window_guest_list_num_pages = 1 + (numGuests - 1) / GUESTS_PER_PAGE;
```

Comment:
```suggestion
            _window_guest_list_num_pages = ((numGuests - 1) / GUESTS_PER_PAGE) + 1;
```

Reply:
Personally, I think the original is clearer. There is, after all, always one page.

Reply:
Meh, not sure if I agree, but it's too minor to argue about. I'll merge it as-is.

Reply:
I would agree with reversing the order but I don't think that the redundant parentheses add much to clarity, or was there a different motivation?